### PR TITLE
made product tour and my-topics work with new concepts API

### PR DESCRIPTION
**Pull request by Lrnk**
a bit sad about losing the stream pun

### CODE REVIEW

File: server/jobs/hot-topics.js [hunk 1]
```diff
@@ -1,40 +1,37 @@
...
+const topBrands = require('../models/top-brands');
+const topAuthors = require('../models/top-authors');
+
+let hotTopics = {};
+
+//adapt for new concepts API results schema
+function adaptTags (rawTags) {
+	return rawTags.map(rawTag => ({
+		id: rawTag.idV1 || rawTag.id,
+		name: rawTag.prefLabel || rawTag.name,
```
Comment (adgad) [inline server/jobs/hot-topics.js]: Is there any chance this would get a weird mix of v1 ID and v2 name? (which in turn means MERGEs on the DB would fail?)

**Comment by adgad** [comment]
LGTM

File: server/jobs/hot-topics.js (hunk 1 above)
Comment (Lrnk) [inline server/jobs/hot-topics.js]: Ah yeah that's an interesting consideration. There's no chance of this occurring with the APIs as they are currently, so as long as its versioned properly, we should be ok

**Comment by Lrnk** [comment]
thank you!

**Pull request merged**